02-peer-protocol.md

```diff
@@ -980,7 +980,7 @@ last `commitment_signed` message the receiving node has sent, it
 SHOULD fail the channel.
 
 If `next_remote_revocation_number` is equal to the commitment number of
-the last `revoke_and_ack` the receiving node has sent, it MUST re-send
+the last `revoke_and_ack` the receiving node has sent and the receiving node has not already received a `closing_signed`, it MUST re-send
```

Comment:
As you state in the rationale below, the sender won't send a `closing_signed` until it has sent the last `revoke_and_ack`.

Thus I don't see how this scenario is possible?